src/libs/providers.js

```diff
@@ -1,5 +1,5 @@
 export const allProviders = [
-  { key: 'fence', name: 'NHLBI BioData Catalyst Framework Services' },
-  { key: 'dcf-fence', name: 'NCI CRDC Framework Services' },
-  { key: 'anvil', name: 'NHGRI AnVIL Data Commons Framework Services' }
+  { key: 'fence', name: 'NHLBI BioData Catalyst Framework Services', short: 'NHLBI' },
```

Comment:
These could be changed to something else if it makes more sense...

Reply:
I think this makes sense to me. We can change them down the road if it needs to change.